keras_retinanet/models/mobilenet.py

```diff
@@ -0,0 +1,56 @@
+import keras
+from keras import backend as K
+from keras.models import Model
+from keras.applications.mobilenet import MobileNet, BASE_WEIGHT_PATH, get_file
+
+from ..models import retinanet
+
+custom_objects = retinanet.custom_objects.copy()
+
+
+def download_imagenet(backbone):
+    alpha = 1.0
+    rows = backbone
+
+    # load weights
+    if K.image_data_format() == 'channels_first':
+        raise ValueError('Weights for "channels_last" format '
+                         'are not available.')
+    if alpha == 1.0:
+        alpha_text = '1_0'
+    elif alpha == 0.75:
+        alpha_text = '7_5'
+    elif alpha == 0.50:
+        alpha_text = '5_0'
+    else:
+        alpha_text = '2_5'
+
+    model_name = 'mobilenet_%s_%d_tf_no_top.h5' % (alpha_text, rows)
+    weigh_path = BASE_WEIGHT_PATH + model_name
+    weights_path = get_file(model_name,
+                            weigh_path,
+                            cache_subdir='models')
+
+    return weights_path
+
+
+def mobilenet_retinanet(num_classes, backbone=224, inputs=None, **kwargs):
+    alpha = 1.0
```

Comment:
Should this be configurable? I haven't read much on MobileNet yet but from what I remember this defines the complexity of the model?

Reply:
Yes, they should be configurable. But before doing that we should refactor the backbone initialisation and consider that backbones could have different parameters.

Reply:
Did some work on this in the latest commit.